pennylane/gradients/finite_difference.py

```diff
@@ -323,8 +323,19 @@ def finite_diff(
         This can be useful if the underlying circuits representing the gradient
         computation need to be analyzed.
 
-        The output tapes can then be evaluated and post-processed to retrieve
-        the gradient:
+        Note that ``argnum`` refers to the index of a parameter within the list of trainable

```

Comment:
Minor: Personally I would prefer to have a better description of `argnum` in the docs rather than an example.